R/compute_age_years.R

```diff
@@ -0,0 +1,67 @@
+#' Compute Age in Years
+#'
+#' Converts a set of age values from the specified time unit to years.
+#'
+#' @param age The ages to convert.
+#'
+#'   A numeric vector is expected.
+#'
+#' @param age_unit Age unit.
+#'
+#'   Either a string containing the time unit of all ages in `age` or a character
+#'   vector containing the time units of each age in `age` is expected. Note that
+#'   permitted values are cases insensitive (e.g. 'YEARS' is treated the same
+#'   as 'years' and 'Years').
+#'
+#'   Permitted Values: 'years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds'.
+#'
+#' @details Returns a numeric vector of ages in years. Note, underlying computations
+#' assume an equal number of days in each year (365.25).
+#'
+#' @return The ages contained in `age` converted to years.
+#'
+#' @keywords com_date_time
+#'
+#' @family com_date_time
+#'
+#' @export
+#'
+#' @examples
+#' compute_age_years(
+#'   age = c(240, 360, 480),
+#'   age_unit = "MONTHS"
+#' )
+#'
+#' compute_age_years(
+#'   age = c(28, 1461, 10227),
```

Comment:
Could you add a value which results in a non integer result?

Reply:
Done.